### PR TITLE
All: Partial terminus handling for stop schedule

### PR DESCRIPTION
**Pull request by azime**
- Corrections of partial_terminus for stop_schedule API.
- Tests Added
- Jira: http://jira.canaltp.fr/browse/NAVITIAII-1767

### CODE REVIEW

File: source/jormungandr/tests/departure_board_tests.py [hunk 1]
```diff
@@ -140,6 +140,48 @@ def test_on_datetime(self):
         assert response["stop_schedules"][0]["stop_point"]["id"] == "stop1"
         assert response["stop_schedules"][0]["route"]["line"]["id"] == "line:A"
 
+    def test_partial_terminus(self):
+        """
+        departure board for a given date
```
Comment (antoine-de) [inline source/jormungandr/tests/departure_board_tests.py]: you should update the comment no ?

**Comment by azime** [comment]
corrections

**Comment by azime** [comment]
merge

File: source/time_tables/departure_boards.cpp [hunk 2]
```diff
@@ -181,6 +166,14 @@ departure_board(const std::string& request,
...
             if (! tmp.empty()) {
                 stop_times.insert(stop_times.end(), tmp.begin(), tmp.end());
+            } else {
+                if (stop_point == jpp->journey_pattern->journey_pattern_point_list.back()->stop_point){
+                    if(jpp->stop_point->stop_area == route->destination){
+                        response_status[route->idx] = pbnavitia::ResponseStatus::terminus;
+                    }else{
+                        response_status[route->idx] = pbnavitia::ResponseStatus::partial_terminus;
+                    }
+                }
```
Comment (antoine-de) [inline source/time_tables/departure_boards.cpp:176]: else no tag ?

**Pull request merged**